py/server/deephaven/table.py

```diff
@@ -1842,6 +1842,16 @@ def format_row_where(self, cond: str, formula: str) -> Table:
         except Exception as e:
             raise DHError(e, "failed to color format rows conditionally.") from e
 
+    def format_databar(self, columnName: str, valueColumn: str, min: float, max: float, axis: str = "proportional",
```

Comment:
As discussed in chat, I think you should default to `=None` for min/max, so you can use the value if specified, otherwise you just implicitly use the min/max from that column.
Also, I don't think you need `columnName` here? Just a param named `column` should be fine, since we're just formatting one column:
```suggestion
    def format_databar(self, column: str, min: float = None, max: float = None, axis: str = "proportional",
```

Reply:
I thought they should be different for if the user wants the actual data bar column to be a different name.

Reply:
I think the same - we have the value display as part of the data bar. If they want to display it in a different column for some reason, then they should just copy the column (using a custom column).

Reply:
How would custom columns allow you to format string column A with a databar using ints from column B? Which is something heatmap does allow.

Reply:
I'm confused. How does that apply to databar? Are we displaying a databar alongside the string value from column A?

Reply:
That's a reasonable argument. We can skip having a separate value column, and require the data values and the column being formatted to just be column=